src/components/listview/listview.js

```diff
@@ -76,12 +76,13 @@ import ServerConnections from '../ServerConnections';
         return '';
     }
 
-    function getImageUrl(item, width) {
+    function getImageUrl(item, size) {
         const apiClient = ServerConnections.getApiClient(item.ServerId);
         let itemId;
 
         const options = {
-            maxWidth: width,
+            fillWidth: size,
+            fillHeight: size,
```

Comment:
Couldn't this result in images being scaled too small when the width is less than the height assuming the width is still what is being passed here?

Reply:
listview is the small square one, what this is saying is «Scale this image down, but not under this width and height»
![image](https://user-images.githubusercontent.com/638706/114178416-fe186b00-993d-11eb-86aa-88c8eb596ead.png)
_(screenshot from current release, not this PR)_

If I ask for fillWidth=1, fillHeight=500, the height will be 500 (assuming the source image is larger), and width will be whatever is needed to keep the ratio.

The browser does the cropping (as opposed to scale up then crop which is what is done now if the ratio is weird – like a banner style wide image here for example)

Reply:
Right, but here and below you are using the same value for width and height. So in your example of width=1 you are requesting fillWidth=1 and fillHeight=1.

If these are only used for square images that makes sense, just wanted to check because it's not immediately clear.

Reply:
as far as I can tell, the images requested in listview.js are always displayed in a square, yea